backend/src/journey/journey.service.ts

```diff
@@ -0,0 +1,77 @@
+import { Inject, Injectable } from "@nestjs/common";
+import {
+  HafasClient,
+  Journeys,
+  JourneysOptions,
+  JourneyWithRealtimeData,
+  Location,
+  LocationsOptions,
+  RefreshJourneyOptions,
+  Station,
+  Stop
+} from "hafas-client";
+import { HAFAS_CLIENT, HAFAS_LANGUAGE } from "../symbols";
+import { JourneyRequest } from "./entities/journey.request.entity";
+
+/** Type alias for results of location query. */
+export type Locations = ReadonlyArray<Station | Stop | Location>;
+
+/**
+ * Service that uses HAFAS to plan journeys.
+ */
+@Injectable()
+export class JourneyService {
+
+  constructor(
+    @Inject(HAFAS_CLIENT) private hafas: HafasClient
+  ) { }
+
+
+  /**
+   * Plans and returns options for the requested journey.
+   *
+   * @param journeyRequest request for a journey
+   * @return different options to plan the journey
+   */
+  planJourney(journeyRequest: JourneyRequest): Promise<Journeys> {
+    const options: JourneysOptions = {
+      results: 3,
+      language: HAFAS_LANGUAGE,
+    };
+
+    return this.hafas
+      .journeys(journeyRequest.from, journeyRequest.to, options);
```

Comment:
How to deal with errors that could get thrown by hafas-client? At the moment they are not catched in backend.

Reply:
Maybe discuss this later? So we can get this PR merged and continue with our other issues...

Reply:
see #156